src/GenAPI/Microsoft.DotNet.GenAPI.Tool/Program.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+
+namespace Microsoft.DotNet.GenAPI.Tool
+{
+    /// <summary>
+    /// CLI frontend for the Roslyn-based GenAPI.
+    /// </summary>
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            // Global options
+            Option<string[]> assembliesOption = new("--assemblies",
```

Comment:
Singular option name:

```suggestion
            Option<string[]> assembliesOption = new("--assembly",
```